src/pages/en/guides/server-side-rendering.md

```diff
@@ -132,3 +139,75 @@ export async function get({ params }) {
   });
 }
 ```
+
+While in this example, a valid HTML code can be returned to render the page or it's content, API Routes can be used as REST API Endpoints.
```

Comment:
I think your intention here is to "introduce" the next section which will deal with REST API Endpoints, but I think it's cleaner to just describe each idea in its own section.

Can you combine the idea of returning a valid HTML code to render the page with the sentence above the example, or separate it out into its on sentence here?

Then, mention something about API Routes can be used as REST API endpoints *in* the section below.